unity-game/Assets/Scripts/ElevatorScripts/ElevatorBookends.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class ElevatorBookends : MonoBehaviour {
+
+    public bool leftWall;
+    public bool rightWall;
+    public bool startLeftBarrier;
+    public bool startRightBarrier;
+    public bool endLeftBarrier;
+    public bool endRightBarrier;
```

Comment:
Oh, because the barriers are completely separate in function than the walls. The walls just prevent players from falling off while the elevator is moving - the barriers prevent it when the player isn't on the platform (such as if it's an auto platform, or if they ride an elevator one way, and then take a ladder or something else down to reach the original position.